packages/client/navigation/index.tsx

```diff
@@ -742,6 +835,64 @@ function GuestStackScreen() {
                     headerShadowVisible: false,
                 }}
             />
+            <GuestStack.Screen
+                name="ResetPasswordEmailScreen"
+                component={ResetPasswordEmailScreen}
+                options={{
+                    title: "",
+                    headerTintColor: "#404040",
+                    headerShadowVisible: false,
+                }}
```

Comment:
Add custom back button icon in top nav
```suggestion
                options={({ navigation }) => ({
                    title: "",
                    headerTintColor: "#404040",
                    headerShadowVisible: false,
                    headerLeft: () => BackButton(navigation),
                }}
```